pkg/pool-weighted/test/foundry/WeightedPool8020Factory.t.sol

```diff
@@ -92,7 +106,7 @@ contract WeightedPool8020FactoryTest is WeightedPoolContractsDeployer, VaultCont
         _createPool(tokenA, tokenB);
 
         // Should not be able to deploy identical pool
-        vm.expectRevert("DEPLOYMENT_FAILED");
+        vm.expectRevert();
```

Comment:
Using OZ will give you a better reason if I'm not mistaken.

Let's try to avoid generic expect reverts in the tests.